tests/detect-icmp-seq/test.yaml

```diff
@@ -0,0 +1,18 @@
+# *** Add configuration here ***
+
```

Comment:
Can you please remove these? They're from the createst script, and are not needed ;)